docs/articles/documentation/reference/test-api/test/httpauth.md

```diff
@@ -0,0 +1,16 @@
+---
+layout: docs
+title: Test.httpAuth Method
+permalink: /documentation/reference/test-api/test/httpauth.html
+---
+# Test.httpAuth Method
+
+Specifies the credentials for HTTP [Basic](https://en.wikipedia.org/wiki/Basic_access_authentication) and [Windows (NTLM)](https://en.wikipedia.org/wiki/Integrated_Windows_Authentication) authentication.
+
+```text
+test.httpAuth( credentials )
+```
+
+{% include authentication/http-credentials.md %}
+
+TestCafe uses the specified credentials for all requests that require authentication in the test. To specify credentials to be used for all tests in a fixture, use the [fixture.httpAuth](../fixture/httpauth.md) method.
```

Comment:
Can we change "TestCafe uses the specified credentials" to "TestCafe uses specific credentials"

For the second sentence, please change it to:
To specify credentials for all tests in a fixture, use the [fixture.httpAuth](../fixture/httpauth.md) method.

Reply:
Similar to the previous case, can this be:

*TestCafe uses credentials specified with `test.httpAuth` for all requests that require authentication in the test. To specify credentials for all tests in a fixture, use the `fixture.httpAuth` method*

Reply:
Okay, thank you